src/k8s-extension/HISTORY.rst

```diff
@@ -3,81 +3,33 @@
 Release History
 ===============
 
-0.1.0
-++++++++++++++++++
-* Initial release.
-
-0.1.1
-++++++++++++++++++
-* Add support for microsoft-azure-defender extension type
-
-0.1.2
-++++++++++++++++++
-
-* Add support for Arc Appliance cluster type
-
-0.1.3
-++++++++++++++++++
-
-* Customization for microsoft.openservicemesh
-
-0.1PP.4
+0.2.0
```

Comment:
Since 0.3.0 is the new release, can you update this file to include that version?

Reply:
Are we skipping 0.2.0? As far as I'm aware, we haven't released a version 0.2.0 of `k8s-extension`